salesagility/SuiteCRM
# Fixed #1189 - AOR - Report doesn't display Image for image fieldtype

## Description



Fixes broken images on reports:

![screenshot from 2016-04-01 10 45 57](https://cloud.githubusercontent.com/assets/13675281/14203608/f789fe22-f7f6-11e5-9baf-dc7883330945.png)

## Motivation and Context

Issue reference: #1189 

## How To Test This

1. Create a report on a field that contains an image. 
2. Check that the image is properly displayed.

## Types of changes


### Final checklist
